File: src/main/java/com/rkroom/blog/service/CategoryService.java
package com.rkroom.blog.service;

import com.rkroom.blog.entity.Categories;

import java.util.List;

public interface CategoryService {
    List<Categories> selectAll();
    void insert(Categories category);
    Categories selectById(int id);
    List selectCategoryByIsindex(boolean status);
    void changeIsindexById(boolean status,int id);
}

File: src/main/java/com/rkroom/blog/service/CategoryService.kt
package com.rkroom.blog.service

import com.rkroom.blog.entity.Categories

interface CategoryService {
    fun selectAll(): List<Categories?>?
    fun insert(category: Categories?)
    fun selectById(id: Int): Categories?
    fun selectCategoryByIsindex(status: Boolean): List<*>?
    fun changeIsindexById(status: Boolean, id: Int)
}